applications/inquiry_proposal_applicants/FoxGaming.md

```diff
@@ -0,0 +1,84 @@
+##### ***EGF Proposal Form (FoxGaming)***
+
+## ***Overview***
+
+**The project name**: Fox Gaming
+
+**Summary of your project and how it would help our ecosystem:** NFT-Marketplace is a must have every blockchain should have to attract new investors/users. Having it will increase transactions, users and  fee´s on the fx blockchain. 
+
+**Team Leadername:** Loil -Fox Gaming
+
+**Payment address:**  fx1780uflqpght7etzq8auv0ram6rwy3wd6dasex9 f(x)Core (Function X Mainnet), $FX
+
+**The level of amount you are requesting for? Level 1,2, 3, 4:** Level 3 50,000 USD in FX tokens
+
+**Distribution schematic:**
+
+1. 25,000 USD after the proposal passes (hiring known NFT Artists to kick start the marketplace)
+2. 25,000 USD split into five tranches each 5,000USD. After significant development and improvement as:
+
+(nft launchpad, marketplace design 2.0 , daily user milestone perhaps 100-1000 daily users and so on)
+
+We apply to the three-person council (fx foundation) to release the future tranchesb after we can prove the improvements.
+
+**Category your project belongs to:** NFT Marketplace
+
+
+## ***Project Description***
+
+See all hands meeting and forum:
+
+https://www.youtube.com/watch?v=CLCGdvxwM0E
+
+https://forum.functionx.io/t/foxgaming-proposal-our-preview-discussion/2757/81
+
+## ***Team profile***
+
+**More about the team:**
+
+**Thilo Colberg**, 30 years old living in germany. Owner of two financial companys and developer/owner. 
+https://de.linkedin.com/in/thilo-colberg-4b36b61b3
+
+**Jose Alberto Marquez**, 39 years old living in mexico. Chief Information Security Officer, Data Protection Officer〈 Business Model Innovation,Enterprise Architecture, IoT 
+& AI,  Researcher, Author & Speaker, SAP veteran.
+https://www.linkedin.com/in/jomarq/
+
+**Jan Matthias Colberg**, 32 years old living in kuwait city. Author and Game Full Stack Online Marketing & Digital Project Lead. 
+https://kw.linkedin.com/in/jan-matthias-colberg-98a8a7b1
+
+**Zakry Cami**, 45 years old living in Paris. Full Time Blockchain Developer and owner of an IT solution company. 
+
+Website: https://officialfoxcoin.com or https://foxchange.io
+
+Twitter: https://twitter.com/officialfoxcoin
+
+Telegram: https://t.me/foxcoinholeofficial
+
+Registered Address: Germany, 69115 Heidelberg, Langer Anger 197
+
+Contact Email: ceo@officialfoxcoin.com
+
+## ***Ecosystem Fit/Impact***
+ 
+**What problem are you solving:**
+
+Our goal is to provide some serious upgrade to function-x blockchain with a mass adoption use case!
+Every chain needs its own NFT marketplace, so here is our! FoxChange where people can sell Function X NFT & more! 
+
+**Who will benefit:**
+
+The entire FunctionX ecosystem. The most important usecase is the fee`s people have to pay in FX when they make any transaction on our marketplace or move the coins NFT´S attract a lot of people and investors.
```

Comment:
would you consider airdrops or like free usage of your platform for say a month. or is there something that will immediately benefit the community? something tangible?